src/sentry/static/sentry/app/views/app/index.tsx

```diff
@@ -147,7 +153,7 @@ class App extends React.Component {
       }
 
       // Otherwise, the user has become unauthenticated. Send them to auth
-      Cookies.set('session_expired', 1);
+      Cookies.set('session_expired', '1');
```

Comment:
On backend we don't do a strict equality comparison, so this should be fine (well we do in tests, but it correctly compares w/ a string).

Reply:
Cookie values are always strings as well.